### PR TITLE
Upload coverage information to Firebase

### PR DESCRIPTION
**Pull request by robertsipka**

### CODE REVIEW

File: API/common/utils.py [hunk 1]
```diff
@@ -538,3 +546,61 @@ def read_port_from_url(url):
...
+
+
+def parse_coverage_info(env, coverage_output):
+    '''
+    Parse and create coverage information
+    '''
+    coverage_info = {}
+
+    iotjs_dir = env['modules']['iotjs']['src']
+    js_folder = join(iotjs_dir, 'src/js/')
```
Comment (rtakacs) [inline API/common/utils.py]: I would create an entry in the `resources.json`, iotjs['path'] section.

File: API/common/utils.py [hunk 2]
```diff
@@ -538,3 +546,61 @@ def read_port_from_url(url):
...
+        coverage_info[filename]['coverage'] = [0, 0]
+
+        js_file_path = join(js_folder, js_file)
+
+        with open(js_file_path, "r") as js_source:
+            lines = js_source.readlines()
+
+            for _, val in enumerate(lines):
+                # '0' indicates that the line has not been reached yet.
+                coverage_info[filename]['lines'].append([val, '0'])
```
Comment (rtakacs) [inline API/common/utils.py]:
```py
for line in lines
```

**Comment by robertsipka** [comment]
@rtakacs : Thanks, I've updated the patch.

**Comment by bzsolt** [comment]
LGTM

**Pull request merged**